src/App.js

```diff
@@ -10,7 +10,11 @@ export class App extends Component {
   };
 
   addMovie = (movie) => {
-    // put your code here
+    this.setState(state => (
```

Comment:
Краще явно вказувати, що ти береш значення **попереднього** стейта.
```suggestion
    this.setState(prevState => (
```

Reply:
fixed